### DISCUSSION TITLE
show argument error

### DISCUSSION THREAD

**Discussion by ahadalichowdhury** [projectdiscovery/httpx#786] [Q&A] [answered]
I install httpx using go . after install httpx , i want to run httpx -h in terminal it's show
Error: Option '-h' requires 2 arguments.

How can i solve this problem??

**Comment by forgedhallpass** [comment, marked as answer]
See this comment and the related links in it: https://github.com/projectdiscovery/httpx/discussions/774#discussioncomment-3673676

**Comment by ahadalichowdhury** [comment]
is this possible to open httpx online httpx without $GOPATH variable??? as like as subzy??

**Reply by forgedhallpass** [reply]
What do you mean by "online"? The `$GOPATH` is only relevant if you install it using the `go install` command, and it's specific to `go`.

If you want, you can go to releases,  download a binary and simply copy it to some directory within you `PATH`, e.g. `/usr/local/bin`.
